arm9/modules/28/asm/module_28.s

```diff
@@ -2965,27 +2965,87 @@ _02255E74: .word 0x00000163
+	.word MOD28_02255188
+
+	.global MOD28_02255EA8
+MOD28_02255EA8: ; 0x02255EA8
+	.word MOD28_022553B4, MOD28_0225549C, MOD28_022555B4, MOD28_02255690
+	.word MOD28_02255768, MOD28_0225580C
+
+	.global MOD28_02255EC0
+MOD28_02255EC0: ; 0x02255EC0
+	.byte 0x30, 0x00, 0x2C, 0x00, 0xB0, 0x00, 0x2C, 0x00
 	.byte 0x30, 0x00, 0x5C, 0x00, 0xB0, 0x00, 0x5C, 0x00, 0x30, 0x00, 0x8C, 0x00, 0xB0, 0x00, 0x8C, 0x00
+
+	.global MOD28_02255ED8
+MOD28_02255ED8: ; 0x02255ED8
 	.byte 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x08, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
-	.byte 0x01, 0x00, 0x0E, 0x00, 0x00, 0x02, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
-	.byte 0x19, 0x4C, 0x25, 0x02, 0x00, 0x00, 0x00, 0x00, 0x01, 0x00, 0x00, 0x00, 0x19, 0x4F, 0x25, 0x02
-	.byte 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
+	.byte 0x01, 0x00, 0x0E, 0x00, 0x00, 0x02, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
+
+	.global MOD28_02255EF4
+MOD28_02255EF4: ; 0x02255EF4
+	.word 0x00, MOD28_02254C18, 0x00
+	.word 0x01, MOD28_02254F18, 0x00
+	.word 0x00, 0x00000000, 0x00
 
 	.section .data
-	; 0x02255F20
-	.byte 0x69, 0x63, 0x6F, 0x6E, 0x5B, 0x25, 0x64, 0x5D, 0x20, 0x52, 0x45, 0x46, 0x5B, 0x25, 0x64, 0x5D
-	.byte 0x0A, 0x00, 0x00, 0x00, 0x2D, 0x2D, 0x2D, 0x2D, 0x2D, 0x2D, 0x2D, 0x2D, 0x2D, 0x2D, 0x2D, 0x2D
-	.byte 0x0A, 0x00, 0x00, 0x00, 0x20, 0x69, 0x63, 0x6F, 0x6E, 0x5B, 0x25, 0x64, 0x5D, 0x20, 0x44, 0x65
-	.byte 0x66, 0x61, 0x75, 0x6C, 0x74, 0x0A, 0x00, 0x00, 0x20, 0x69, 0x63, 0x6F, 0x6E, 0x5B, 0x25, 0x64
-	.byte 0x5D, 0x20, 0x52, 0x65, 0x61, 0x4C, 0x69, 0x6B, 0x65, 0x0A, 0x00, 0x00, 0x20, 0x69, 0x63, 0x6F
-	.byte 0x6E, 0x5B, 0x25, 0x64, 0x5D, 0x20, 0x52, 0x65, 0x61, 0x48, 0x61, 0x74, 0x65, 0x0A, 0x00, 0x00
-	.byte 0x20, 0x69, 0x63, 0x6F, 0x6E, 0x5B, 0x25, 0x64, 0x5D, 0x20, 0x54, 0x63, 0x67, 0x4C, 0x69, 0x6B
-	.byte 0x65, 0x0A, 0x00, 0x00, 0x20, 0x69, 0x63, 0x6F, 0x6E, 0x5B, 0x25, 0x64, 0x5D, 0x20, 0x54, 0x63
-	.byte 0x68, 0x48, 0x61, 0x74, 0x65, 0x0A, 0x00, 0x00, 0x20, 0x69, 0x63, 0x6F, 0x6E, 0x5B, 0x25, 0x64
-	.byte 0x5D, 0x20, 0x52, 0x65, 0x73, 0x65, 0x74, 0x21, 0x21, 0x0A, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
+
+	.global MOD28_02255F20
+MOD28_02255F20: ; 0x02255F20
+	.asciz "icon[%d] REF[%d]\n"
+
+	.balign 4
+	.global MOD28_02255F34
+MOD28_02255F34: ; 0x02255F34
+	.asciz "------------\n"
+
+	.balign 4
+	.global MOD28_02255F44
+MOD28_02255F44: ; 0x02255F44
+	.asciz " icon[%d] Default\n"
+
+	.balign 4
+	.global MOD28_02255F58
+MOD28_02255F58: ; 0x02255F58
+	.asciz " icon[%d] ReaLike\n"
+
+	.balign 4
+	.global MOD28_02255F6C
+MOD28_02255F6C: ; 0x02255F6C
+	.asciz " icon[%d] ReaHate\n"
+
+	.balign 4
+	.global MOD28_02255F80
+MOD28_02255F80: ; 0x02255F80
+	.asciz " icon[%d] TcgLike\n"
+
+	.balign 4
+	.global MOD28_02255F94
+MOD28_02255F94: ; 0x02255F94
+	.asciz " icon[%d] TchHate\n"
+
+	.balign 4
+	.global MOD28_02255FA8
+MOD28_02255FA8: ; 0x02255FA8
+	.asciz " icon[%d] Reset!!\n"
+	.byte 0x00, 0x00, 0x00, 0x00, 0x00
```

Comment:
Trailing `.byte 0`s can be omitted as they are filled in by alignment (to 0x20).